modules/storage/storage-account/main.bicep

```
@@ -55,6 +151,30 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' = {
   }
 }
 
+resource blobService 'Microsoft.Storage/storageAccounts/blobServices@2022-09-01' = if (enableBlobService) {
+  name: 'default'
+  parent: storageAccount
+  properties: blobProperties
+}
+
+resource container 'Microsoft.Storage/storageAccounts/blobServices/containers@2022-09-01' = if (enableBlobService) {
+  name: blobContainerName
+  parent: blobService
+  properties: blobContainerProperties
+}
+
+module storageRbac '.bicep/nested_rbac.bicep' = [for (roleAssignment, index) in roleAssignments: {
+  name: '${uniqueString(deployment().name, location)}-acr-rbac-${index}'
```

```suggestion
  name: 'acr-rbac-${index}-${uniqueString(deployment().name, location)}'
```

This will be a little easier to read in the logs.

---

Updated in https://github.com/Azure/bicep-registry-modules/commit/06b88359be80e90b6d23b5abd98746a7205b69e8